<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MyselfWidget</class>
 <widget class="QWidget" name="MyselfWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1341</width>
    <height>784</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>7</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="baseInfoArea" native="true">
     <property name="minimumSize">
      <size>
       <width>1310</width>
       <height>120</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>1310</width>
       <height>120</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">#baseInfoArea{
    background-color : #F5F5F5;
}</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>24</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>60</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QWidget" name="userInfoBox" native="true">
        <property name="minimumSize">
         <size>
          <width>808</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>808</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <property name="spacing">
          <number>8</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="AvatarButton" name="avatarBtn">
           <property name="minimumSize">
            <size>
             <width>60</width>
             <height>60</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>60</width>
             <height>60</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">#avatarBtn{
    background-color : transparent;
    border : none;
    border-radius : 30px;
    border-image : url(&quot;:/images/myself/defaultAvatar.png&quot;);
}</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="iconSize">
            <size>
             <width>60</width>
             <height>60</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="MyPushButton" name="nicknameBtn">
           <property name="styleSheet">
            <string notr="true">#nicknameBtn{
    border : none;
    font-size : 18px;
    font-family : 微软雅黑;
}</string>
           </property>
           <property name="text">
            <string>比特视频用户-123456</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="settingBtn">
           <property name="minimumSize">
            <size>
             <width>54</width>
             <height>17</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>54</width>
             <height>17</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">#settingBtn{
    border : none;
    border-image : url(&quot;:/images/myself/xiugai.png&quot;);
}</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Orientation::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>545</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="videoNumBox" native="true">
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <layout class="QGridLayout" name="gridLayout">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>31</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>31</number>
         </property>
         <property name="horizontalSpacing">
          <number>7</number>
         </property>
         <property name="verticalSpacing">
          <number>14</number>
         </property>
         <item row="0" column="0">
          <widget class="QLabel" name="attentionLabel">
           <property name="styleSheet">
            <string notr="true">#attentionLabel{
     font-family : 微软雅黑;
     font-size : 16px;
     color : #999999;
}</string>
           </property>
           <property name="text">
            <string>关注数</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignmentFlag::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLabel" name="fansLabel">
           <property name="styleSheet">
            <string notr="true">#fansLabel{
     font-family : 微软雅黑;
     font-size : 16px;
     color : #999999;
}</string>
           </property>
           <property name="text">
            <string>粉丝数</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignmentFlag::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QLabel" name="likeLabel">
           <property name="styleSheet">
            <string notr="true">#likeLabel{
     font-family : 微软雅黑;
     font-size : 16px;
     color : #999999;
}</string>
           </property>
           <property name="text">
            <string>点赞数</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignmentFlag::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="0" column="3">
          <widget class="QLabel" name="playLabel">
           <property name="styleSheet">
            <string notr="true">#playLabel{
     font-family : 微软雅黑;
     font-size : 16px;
     color : #999999;
}</string>
           </property>
           <property name="text">
            <string>播放数</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignmentFlag::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="attentionCountLabel">
           <property name="styleSheet">
            <string notr="true">#attentionCountLabel{
    font-family : 微软雅黑;
    font-size : 18px;
    color : #222222;
}</string>
           </property>
           <property name="text">
            <string>123</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignmentFlag::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLabel" name="fansCountLabel">
           <property name="styleSheet">
            <string notr="true">#fansCountLabel{
    font-family : 微软雅黑;
    font-size : 18px;
    color : #222222;
}</string>
           </property>
           <property name="text">
            <string>123</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignmentFlag::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="1" column="2">
          <widget class="QLabel" name="likeCountLabel">
           <property name="styleSheet">
            <string notr="true">#likeCountLabel{
    font-family : 微软雅黑;
    font-size : 18px;
    color : #222222;
}</string>
           </property>
           <property name="text">
            <string>1.23w</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignmentFlag::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="1" column="3">
          <widget class="QLabel" name="playCountLabel">
           <property name="styleSheet">
            <string notr="true">#playCountLabel{
    font-family : 微软雅黑;
    font-size : 18px;
    color : #222222;
}</string>
           </property>
           <property name="text">
            <string>1.23w</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignmentFlag::AlignCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="btnBox" native="true">
        <property name="minimumSize">
         <size>
          <width>94</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>94</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <widget class="AttentionButton" name="attentionBtn">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>41</y>
           <width>94</width>
           <height>38</height>
          </rect>
         </property>
         <property name="text">
          <string>关注</string>
         </property>
        </widget>
        <widget class="QPushButton" name="quitBtn">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>54</y>
           <width>94</width>
           <height>30</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">#quitBtn{
    border : 1px solid #DDDDDD;
    border-radius : 15px;
    font-family : 微软雅黑;
    color : #999999;
    font-size : 14px;
}</string>
         </property>
         <property name="text">
          <string>退出登录</string>
         </property>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="titleBar" native="true">
     <property name="minimumSize">
      <size>
       <width>1310</width>
       <height>76</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>1310</width>
       <height>76</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="myVideoLabel">
        <property name="styleSheet">
         <string notr="true">#myVideoLabel{
    font-family : 微软雅黑;
    font-size : 18px;
}</string>
        </property>
        <property name="text">
         <string>我的视频</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Orientation::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>1110</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="uploadVideoBtn">
        <property name="minimumSize">
         <size>
          <width>116</width>
          <height>30</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>116</width>
          <height>30</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">#uploadVideoBtn{
    border-radius : 15px;
    border-image : url(&quot;:/images/myself/shangchuanshipin.png&quot;);
}</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="videoInfoArea" native="true">
     <property name="minimumSize">
      <size>
       <width>1340</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>1340</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QScrollArea" name="scrollArea">
        <property name="styleSheet">
         <string notr="true">#scrollArea{
    border : none;
    background-color : #FFFFFF;
}

QScrollBar::vertical{
    border : none;
    background-color : #FFFFFF;
    width : 10px;
}

QScrollBar::handle:vertical{
    border : none;
    border-radius : 5px;
    background-color : #E8E8E8;
    min-height : 134px;
}

QScroll::up-arrow:vertical, QScrollBar::down-arrow:vertical
{
     border : none;
     width : 0px;
     height : 0px;
}



</string>
        </property>
        <property name="horizontalScrollBarPolicy">
         <enum>Qt::ScrollBarPolicy::ScrollBarAlwaysOff</enum>
        </property>
        <property name="widgetResizable">
         <bool>true</bool>
        </property>
        <widget class="QWidget" name="scrollAreaWidgetContents">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>1340</width>
           <height>574</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">#scrollAreaWidgetContents{
    background-color : #FFFFFF;
}</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QWidget" name="container" native="true">
            <layout class="QVBoxLayout" name="verticalLayout_4">
             <property name="spacing">
              <number>0</number>
             </property>
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <layout class="QGridLayout" name="layout">
               <property name="leftMargin">
                <number>5</number>
               </property>
               <property name="spacing">
                <number>20</number>
               </property>
              </layout>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>AvatarButton</class>
   <extends>QPushButton</extends>
   <header location="global">avatarbutton.h</header>
  </customwidget>
  <customwidget>
   <class>AttentionButton</class>
   <extends>QPushButton</extends>
   <header location="global">myselfwidget.h</header>
  </customwidget>
  <customwidget>
   <class>MyPushButton</class>
   <extends>QPushButton</extends>
   <header location="global">avatarbutton.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
